README.md

```diff
@@ -59,17 +59,18 @@ for more information).
 make               # Build lambdapi.
 make doc           # Build the documentation.
 make install       # Install the program.
+make install_emacs # Install emacs mode, see the note below
 make install_vim   # Install vim support.
 ```
 
 **Note:** you can run `lambdapi` without installing with `dune exec -- lambdapi`.
 
-**Note on Emacs:** `make install` installs the `lambdapi-mode`, an Emacs major
-mode for editing `lambdapi` files in `$(opam var share)/emacs/site-lisp`.
-To load the `lambdapi-mode` automatically when editing `*.lp` files, add `(load
-"lambdapi-site-file")` to your `~/.emacs.d/init.el` or `~/.emacs`.
-If `lambdapi-mode` is not activated when editing a file ending in `.lp`, read
-the installation section of the
+**Note on Emacs:** `make install_emacs` installs the `lambdapi-mode`, an Emacs
```

Comment:
Instead of having a specific note on Emacs only here, wouldn't it be better to mention that some plugins for several editors are provided, with a link to ui.md?

Reply:
OK, and regarding the documentation, I'd rather centralise information, so I'd like to reduce the content of `ui.md` to a link to `emacs.md` (concerning the emacs part), to avoid spreading information, what do you think?

Reply:
Isn't already the case (see https://github.com/Deducteam/lambdapi/pull/425)?

Reply:
Ah nice, I was looking at an older version on my computer, sorry.